src/shared/common/forms/confirmation-with-team-name-and-case-ref.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+export default class ConfirmationWithCaseRef extends Component {
```

Comment:
Also spotted after the initial review... the name of this class needs to be changed and should read `ConfirmationWithTeamNameAndCaseRef`